File: python/prism_sdk/local_instrument_execution_integrity_contract_model.py
"""Lab P32 local contract_model instrument-execution integrity feature F05."""
from .instrument_execution_integrity_support import InstrumentExecutionRequest4,InstrumentExecutionCard7,InstrumentExecutionIntegrityError,manifest,qualify
FEATURE_ID="AFA-lab-P32-F05";CONTRACT_VERSION="lab-local_instrument_execution_integrity_contract_model/1.0"
def local_instrument_execution_integrity_contract_model_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="contract_model")
def qualify_local_instrument_execution_integrity_contract_model(request:InstrumentExecutionRequest4)->InstrumentExecutionCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="contract_model")

File: crates/lab/src/local_instrument_execution_integrity_contract_model.rs
//! Lab P32 local contract_model instrument-execution integrity feature.
use super::instrument_execution_integrity_support::{manifest,qualify,InstrumentExecutionCard7,InstrumentExecutionIntegrityError,InstrumentExecutionRequest4};
pub const FEATURE_ID:&str="AFA-lab-P32-F05";pub const CONTRACT_VERSION:&str="lab-local_instrument_execution_integrity_contract_model/1.0";
pub fn local_instrument_execution_integrity_contract_model_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"local","contract_model")}
pub fn qualify_local_instrument_execution_integrity_contract_model(request:&InstrumentExecutionRequest4)->Result<InstrumentExecutionCard7,InstrumentExecutionIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"local","contract_model")}
